dmapiclient/data.py

```diff
@@ -615,12 +615,13 @@ def is_supplier_eligible_for_brief(self, supplier_id, brief_id):
             params={"supplier_id": supplier_id}
         )['services']) > 0
 
-    def create_brief_response(self, brief_id, supplier_id, data, user):
+    def create_brief_response(self, brief_id, supplier_id, data, user, page_questions=None):
```

Comment:
I guess so. I've looked at how we are doing this for the other API client methods and found at 3 methods that do it this way so I'd be tempted to keep with convention (to be honest I don't see a particular benefit/disadvantage or doing it either way).